debezium-standalone-quarkus/src/test/java/io/debezium/standalone/quarkus/ServerIT.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.standalone.quarkus;
+
+import java.time.Duration;
+
+import javax.enterprise.event.Observes;
+import javax.inject.Inject;
+
+import org.awaitility.Awaitility;
+import org.fest.assertions.Assertions;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.Test;
+
+import io.debezium.standalone.quarkus.events.ConnectorCompletedEvent;
+import io.debezium.standalone.quarkus.events.ConnectorStartedEvent;
+import io.debezium.util.Testing;
+import io.quarkus.test.junit.QuarkusTest;
+
+/**
+ * Integration test that verifies basic reading from PostgreSQL database.
+ *
+ * @author Jiri Pechanec
+ */
+@QuarkusTest
+public class ServerIT {
```

Comment:
Could you add a test that runs against Kinesis? It can (and should) be disabled by default, but would be interesting to see the required config and be easily able to run it against one own's account.